src/cc/arduino/plugins/wifi101/SerialPortListModel.java

```diff
@@ -38,10 +40,16 @@
 class SerialPortListModel implements ListModel<String> {
 
 	private List<BoardPort> ports;
-
+	private HashMap <String,String> boardList;
 	public SerialPortListModel() {
 		ports = Base.getDiscoveryManager().discovery();
 		ports.removeIf(port -> !port.getProtocol().equals("serial"));
+		boardList= new HashMap();
+
+		boardList.put("2341:8054","WifiNINA");
```

Comment:
can we add this to the config file?

Reply:
Would it be possible to retrieve this mapping from IDE source?